jdaviz/configs/specviz2d/plugins/spectral_extraction/spectral_extraction.py

```diff
@@ -442,7 +447,12 @@ def create_bg(self, add_data=False):
         return bg_spec
 
     def vue_create_bg(self, *args):
-        _ = self.create_bg(add_data=True)
+        try:
+            self.create_bg(add_data=True)
+        except Exception as e:
+            msg = (f"Specreduce background failed with the following error: {repr(e)}")
+            msg = SnackbarMessage(msg, color='error', sender=self)
+            self.app.hub.broadcast(msg)
```

Comment:
Nitpick: Is there a reason why we can't do this in one line?
```suggestion
            self.app.hub.broadcast(SnackbarMessage(
                f"Specreduce background failed with the following error: {repr(e)}", color='error', sender=self))
```

Reply:
no, I was just following the pattern that is already used app-wide.  I can change this one instance here if you think we're better off slowly consolidating them rather than addressing them all at once sometime in the future?

Reply:
Since this change is invisible to the user, I am okay to change this one at a time. Thanks!